mlflow/utils/rest_utils.py

```diff
@@ -295,7 +294,6 @@ def call_endpoints(host_creds, endpoints, json_body, response_proto):
                 raise e
 
 
-@contextmanager
```

Comment:
The following code still works because `cloud_storage_http_request` returns a requests' Response object which is a context manager.

```
with cloud_storage_http_request(...):
    ...
```